### ISSUE TITLE
ShadeCN components in VS Code

### ISSUE THREAD

**Issue by xerudro** [oven-sh/bun#12572]
### How can we reproduce the crash?

installed bun on vscode
created app@latest
installed chadecn
added components and the bun got grashed

### Stack Trace (bun.report)

Bun v1.1.20 ([`ae19489`](<https://github.com/oven-sh/bun/tree/ae19489250ab74ee3ea7d5b50fca5d7d2b4e6d66>)) on windows x86_64 [RunCommand]

Segmentation fault at address 0x00000005

- *5 unknown/js code*
- `llint_entry`

**Comment by nektro** [comment]
```sh
❯ C:\bun\build\debug\bun-debug.exe --bun x create-remix@latest my-remix
❯ cd .\my-remix\
❯ C:\bun\build\debug\bun-debug.exe --bun x shadcn@latest init
❯ C:\bun\build\debug\bun-debug.exe --bun x shadcn@latest add alert
✔ Checking registry.
✔ Created 1 file:
  - components\ui\alert.tsx


panic: Segmentation fault at address 0xFFFFFFFFFFFFFFFF
panicked during a panic. Aborting.
```

**Comment by heimskr** [comment]
Fixed by #15357.
